docs/source/reference/modding/settings/game.rst

```diff
@@ -282,3 +282,19 @@ equivalent to the one introduced by the equivalent Morrowind Code Patch feature.
 This makes the movement speed behavior more fair between different races.
 
 This setting can be controlled in Advanced tab of the launcher.
+
+projectiles enchant multiplier
+------------------------------
+
+:Type:		floating point
+:Range:		0.0 to 1.0
+:Default:	0.0
+
+This setting determines which percentage of soulgem charge can be used to enchant multiple projectiles (thrown weapons, arrows and bolts):
+
+count = gemCharge / enchantmentPower * projectilesEnchantMultiplier
+
+Zero value means that you can enchant only one projectile at once.
+If you want to enchant multiple projectiles at once, it is recommended to use value near 0.25 (25% of charge, as in MCP).
```

Comment:
If you want to have Morrowind Code Patch-like count of projectiles being enchanted at once, set this value to 0.25 (i.e. 25% of the charge).